README.md

```diff
@@ -46,6 +46,9 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [XACC](https://github.com/ORNL-QCI/xacc) - Extreme-scale programming model for quantum acceleration within high-performance computing ([arXiv paper](https://arxiv.org/abs/1710.01794)).
 - [qpp](https://github.com/softwareQinc/qpp) - Quantum++ is a modern C++ general purpose quantum computing library, composed solely of template header files.
 
+**Assembly**
```

Comment:
I'm not sure this belongs under "full-stack libraries". It seems to provide simulation routines but there doesn't appear to be any front end? Maybe it belongs under "Quantum simulators" instead? Also, the languages are sorted alphabetically, so "Assembly" should go before "C++".

Reply:
Ok, I'll add it under Quantum Simulators instead, with the "Assembly" heading before "C".

Reply:
I've submitted the updated pull request.